<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppSettings</class>
 <widget class="QWidget" name="AppSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btAdd">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Додати нову програму</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>50</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Редагувати графік виконання</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btRemove">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Видалити з списку</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>641</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>В даному списку знаходяться програми обмежені в часі виконання.</string>
   </property>
  </widget>
  <widget class="QListWidget" name="lstProc">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>681</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Програми</string>
   </property>
  </widget>
  <widget class="QWidget" name="editWidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>681</width>
     <height>451</height>
    </rect>
   </property>
   <widget class="QPushButton" name="btUnselectAll">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>260</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Зняти всі</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>641</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Позначте часові межі в яких програма буде доступна для користувачів.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Вт</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Чт</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Пн</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>641</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string> 0    1     2    3     4     5    6     7     8    9    10   11  12   13   14   15   16  17   18   19   20  21   22   23</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Нд</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btSave">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>260</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Зберегти</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btBack">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>260</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btSelectAll">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>260</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Обрати всі</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Налаштування графіку роботи програми</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Пт</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Сд</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Сб</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btSaveAll">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>50</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Зберегти зміни</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
